client/resources/pixel-buffer.cpp

```diff
@@ -107,6 +113,13 @@ namespace tec {
 					file.write((char*)(p + i), 3); // only output RGB
 				}
 				break;
+			case ImageColorMode::MONOCHROME:
+				break;
```

Comment:
Most of these breaks can be removed as the cases will falldown till exit regardless.